### PR TITLE
Add missing sudo to tee

### PR DESCRIPTION
**Pull request by berendt**
This way it is possible to use the scripts as normal user.

### CODE REVIEW

File: include.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ BOOTSTRAP_LOGFILE="/var/log/install-cloud-in-a-box.log"
 if [[ -n "$BOOTSTRAP_LOGFILE" ]] ;then
    log_ident="$(basename $0|sed '~s,\.sh,,')"
    echo "Logging bootstrap process to $BOOTSTRAP_LOGFILE for $log_ident"
-   exec 1> >(sed "~s,^,$log_ident | ,"|tee -a $BOOTSTRAP_LOGFILE)
-   exec 2> >(sed "~s,^,$log_ident | ,"|tee -a $BOOTSTRAP_LOGFILE >&2)
+   exec 1> >(sed "~s,^,$log_ident | ," | sudo tee -a $BOOTSTRAP_LOGFILE)
```
Comment (scoopex) [inline include.sh:8]: Why is that needed? When installing CIAB, I have not yet noticed that this is needed. Has anything changed at this point?

Comment (berendt) [reply include.sh:8]: It's required when using the script on an existing instance. There you normally want to clone the repository as normal user and want to run the script as a normal user.

Comment (berendt) [reply include.sh:8]: But maybe I only had issues yesterday because I prepare a cloud-in-a-box and started the script again as a normal user?
